src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/ccd/CaseDataHelper.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.sscs.bulkscancore.ccd;
+
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+@Component
+public class CaseDataHelper {
+    private final CoreCaseDataApi coreCaseDataApi;
+
+    @Value("${ccd.case.jurisdictionId}")
+    private String jurisdictionId;
```

Comment:
Is there a reason to mix constructor and field injection?

Reply:
will make it consistent